lib/conditionals.md

```diff
@@ -5,8 +5,19 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional statement is one that allows the computer to run the same code until it runs into a circumstance that has been predefined 
+afterwhich the command differs.  If I were programming a robot this would be a command something like:
+Move forward. If unable to move forward, then turn left. Move forward.
+
 
 # What is the difference between `=` and `==`?
 
-Replace me with your response...
\ No newline at end of file
+To me, these symbols seem to be linked similarly to similies and metaphors
+
+= is for telling a variable to point at an object, kind of like a metaphor. 
+She's a brick house.
+She = brick house
+
+== still maintains that the two inputs are seperate but equal, such as a similie. 
+We're glowing like the metal on the edge of a knife.
+We're glowing == the metal on the edge of a knife
```

Comment:
Just making sure you saw the above comment. I can approve this assignment once you address that question.